src/SchemaManager.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Rbac\Db;
+
+use InvalidArgumentException;
+use Yiisoft\Db\Connection\ConnectionInterface;
+use Yiisoft\Db\Schema\SchemaInterface;
+use Yiisoft\Db\Sqlite\Column;
```

Comment:
Why SQLite? What If I'm use mysql?

Reply:
It's a bug. Thanks for discovering this.